### DISCUSSION TITLE
`GLIBC_2.33' not found on Rasbian OS 6.1.21-v8+

### DISCUSSION THREAD

**Discussion by helo-head** [WiringPi/WiringPi#207] [Q&A]
I am trying to run the latest version (3.1 64 bit) on the  Raspberry PI OS (64-bit bullseye 6.1.21-v8+)  and noted that the required version of the GLIBC is 2.33 while what is installed on bullseye is 2.31-13. I opted to load bullseye vs bookworm as I understood there are issues with bookworms upgraded kernel (6.6). 

Are there any options I'm missing?

For reference Ive included both the error and whats installed 

**Error:** 
$ gpio -v
gpio: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by gpio)
gpio: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by gpio)
gpio: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /lib/libwiringPi.so)
gpio: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /lib/libwiringPi.so)

**Installed:**
uname -ra
Linux RPI-PIHOLE-2 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux

sudo dpkg -l | grep libc6
ii  libc6:arm64                          2.31-13+rpt2+rpi1+deb11u8        arm64        GNU C Library: Shared libraries
ii  libc6-dbg:arm64                      2.31-13+rpt2+rpi1+deb11u8        arm64        GNU C Library: detached debugging symbols
ii  libc6-dev:arm64                      2.31-13+rpt2+rpi1+deb11u8        arm64        GNU C Library: Development Libraries and Header Files

**Comment by mstroh76** [comment]
please build it yourself like shown on the project page 'Install' 'From Source'

**Comment by helo-head** [comment]
Ahhh, said the blind man ...

Seems obvious after the fact ...
